src/studio/src/designer/frontend/shared/components/AltinnRadioGroup.tsx

```diff
@@ -1,21 +1,19 @@
-import { RadioGroup, Typography } from '@material-ui/core';
-import { createStyles, withStyles, WithStyles } from '@material-ui/core/styles';
 import * as React from 'react';
 
-export interface IAltinnRadioGroupComponentProvidedProps extends WithStyles<typeof styles> {
+import { RadioGroup, Typography, makeStyles } from '@material-ui/core';
+
+type AltinnRadioGroupProps = {
```

Comment:
Going with types? Have no issue with it at all, just curious.

Reply:
No, just a habit thats difficult to change 😅 Ill update to interface.

- [ ] change to interface